src/status_im/chat/events/sign_up.cljs

```diff
@@ -107,11 +107,11 @@
   ::contacts-synced
   [re-frame/trim-v (re-frame/inject-cofx :random-id)]
   (fn [{:keys [db random-id] :as cofx} [contacts]]
-    (-> db
+    (-> {:db db}
```

Comment:
Isn't that needlesly complicated ?
The whole thing could be written as:
```clj
(fn [{:keys [random-id] :as cofx} [contacts]]
  (merge (accounts-events/account-update cofx {:signed-up? true})
         {:dispatch (sign-up/contacts-synchronised-event random-id)}))
```

Reply:
I don't know, actually. I see a lot of complicated parts like this one *everywhere* in code. For me it also looks like madness, but I don't really want to refactor it right now.

I agree with all your thoughts, but everything you're talking about is out of the scope of this task.